docs/release-notes/8.10.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[release-notes-header-8.10.0]]
+== 8.10
+
+[discrete]
+[[release-notes-8.10.0]]
+=== 8.10.0
+
+[discrete]
+[[breaking-changes-8.10.0]]
+==== Breaking changes
+There are no breaking changes in 8.10.0.
+
+[discrete]
+[[features-8.10.0]]
+==== New features
+* Adds a component under the Elastic AI Assistant header that allows you to select a different connector ({pull}163666[#163666]).
+* Adds a flyout that allows you to examine rule details when installing or updating a prebuilt rule ({pull}163304[#163304]).
+* Adds the ability to specify custom highlighted fields for an alert ({pull}163235[#163235]).
+* Adds role-based access control privileges for the Elastic AI Assistant ({pull}163031[#163031]).
+* Adds the **Reputation service** option to the malicious behavior protection setting on the Elastic Defend integration policy ({pull}161617[#161617]).
+
+[discrete]
+[[enhancements-8.10.0]]
+==== Enhancements
```

Comment:
Here's another PR that I think was incorrectly labeled as `skip` and is rather a small UX enhancement: https://github.com/elastic/kibana/pull/162477

Reply:
Thanks for catching this and updating the PR labels, @banderror! I added a summary for this PR in [7b5459e](https://github.com/elastic/security-docs/pull/3781/commits/7b5459ef15a2156cb75fe23ada95d5cd2e50f255).